lib/tasks/pact.rake

```diff
@@ -1,19 +1,20 @@
 require "govuk_test"
 
-require "pact/tasks"
-require "pact/tasks/task_helper"
-
 namespace :pact do
   desc "Verify the API contract for a specific branch"
   task "verify:branch", [:branch_name] => :environment do |t, args|
+    begin
+      require "pact/tasks"
+      require "pact/tasks/task_helper"
+    rescue LoadError
+      abort "Pact must be added as a dependency in the downstream app."
+    end
     abort "Please provide a branch name. eg rake #{t.name}[my_feature_branch]" unless args[:branch_name]
   
     pact_version = args[:branch_name] == "master" ? args[:branch_name] : "branch-#{args[:branch_name]}"
-  
-    ClimateControl.modify(GDS_API_ADAPTERS_PACT_VERSION: pact_version) do
-      Pact::TaskHelper.handle_verification_failure do
-        Pact::TaskHelper.execute_pact_verify
-      end
+    ENV["GDS_API_ADAPTERS_PACT_VERSION"] = pact_version
```

Comment:
Just a quick comment before I do a wider review after lunch - I know that we flagged this env var is problematic due to it's GDS_API_ADAPTERS specificity. Is this something we should be dealing with and discussing in this PR or is that for later?

Thanks for doing this by the way 👍

Reply:
For the record, you're referring to [this comment](https://github.com/alphagov/govuk_test/commit/501d7ad98d6e49065d5b9f9be39e5523cc7a31dc#r49160700).

We could rename to `PACT_CONSUMER_VERSION`?

Reply:
Thanks for the detailed review, @kevindew - this is the perfect time to get this right! We're hoping to simplify Pact tests as much as possible, prior to rolling out CD on more Pact-dependent repos.

I think I agree that the `pact:verify:branch` task is not needed. I'm working on removing that from GDS API Adapters in https://github.com/alphagov/gds-api-adapters/pull/1049, and my guinea pig removed task is Frontend's: https://github.com/alphagov/frontend/pull/2712. (I'll later open similar PRs in Collections and Accounts)

It seems to work 🎉 : https://ci.integration.publishing.service.gov.uk/job/gds-api-adapters/job/simplify-pact-verify/3/console

Whilst I'm here, I may rename from `GDS_API_ADAPTERS_PACT_VERSION` to `PACT_CONSUMER_VERSION`, and I may also refactor the additional [Jenkinsfile config](https://github.com/alphagov/gds-api-adapters/blob/b66bb1b6e488b1c01ff08ac2c8f1b5f603d7e4cd/Jenkinsfile#L6-L9) as you pointed out above. Will ping you again when I've made more progress.